Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -5745,7 +5759,7 @@
 				"$(SRCROOT)/Scripts/generate-unified-sources.sh",
 				"$(SRCROOT)/Sources.txt",
 				"$(SRCROOT)/SourcesCocoa.txt",
-				"$(BUILT_PRODUCTS_DIR)/usr/local/include/wtf/Scripts/generate-unified-source-bundles.rb",
+				"$(SRCROOT)../../Source/wtf/Scripts/generate-unified-source-bundles.rb",
```

Comment:
I think this should be:
```suggestion
				"$(SRCROOT)../../Source/WTF/Scripts/generate-unified-source-bundles.rb",
```

Also, what's the reason for changing this? It does seem valid these days (we would never build TestWebKitAPI without a full checkout) but plumbing scripts through WTF's include directory is more typical.

Reply:
I've removed this. I think it was from me trying to work around an error. I didn't realise that it was necessary to build WebKit first because TestWebKitAPI doesn't have correct dependencies to build everything from a clean slate.